topics/certs.txt

```diff
@@ -0,0 +1 @@
+COMING SOON
```

Comment:
Should we document this now or can it wait?

Reply:
Added documentation.